src/main/java/InfrastructureManager/Modules/NetworkStructure/Output/DistanceOutput.java

```diff
@@ -0,0 +1,45 @@
+package InfrastructureManager.Modules.NetworkStructure.Output;
+
+import InfrastructureManager.ModuleManagement.ImmutablePlatformModule;
+import InfrastructureManager.ModuleManagement.PlatformOutput;
+import InfrastructureManager.Modules.NetworkStructure.Network;
+import InfrastructureManager.Modules.NetworkStructure.NetworkModuleObject;
+import InfrastructureManager.Modules.NetworkStructure.Exception.NetworkModuleException;
+
+public class DistanceOutput extends NetworkModuleObject implements PlatformOutput{
+	private Network network;
+	public DistanceOutput(ImmutablePlatformModule module, String name,Network network) {
+		super(module,name);
+		this.network = network;
+	}
+	public void execute(String response) throws NetworkModuleException  {
+		String[] commandLine = response.split(" ");
+		if (commandLine[0].equals("distance")) {
+			try {
+				switch (commandLine[1]) {
+				case "get_physical_distance_devices" -> {
```

Comment:
It would be better if you have the second word as "physical" and a third word as either "devices" or "application". Also why not locations as well?